docs/userguide/usingdocker.md

```diff
@@ -166,15 +166,14 @@ see the application.
 Our Python application is live!
 
 > **Note:**
-> If you have used the `boot2docker` virtual machine on OS X, Windows or Linux,
+> If you have using virtual machine on OS X, Windows or Linux,
 > you'll need to get the IP of the virtual host instead of using localhost.
-> You can do this by running the following outside of the `boot2docker` shell
-> (i.e., from your comment line or terminal application).
+> You can do this by running the `docker-machine ip MACHINE_NAME` from your  command line or terminal application, for example:
```

Comment:
I went with your_vm_name consistently --- it is used in situations where <> can't be used.